tgui/packages/tgui/backend.ts

```diff
@@ -51,7 +58,7 @@ const initialState = {
 export const backendReducer = (state = initialState, action) => {
   const { type, payload } = action;
 
-  if (type === 'backend/update') {
+  if (type === Action.Update) {
```

Comment:
There is no need for constant enums, because actions themselves are constants and contain types, e.g.

```
type === backendUpdate.type
```